components/uniset/src/uniset.rs

```diff
@@ -317,7 +328,15 @@ impl UnicodeSet {
             return false;
         }
         match self.contains_query(from) {
-            Some(pos) => (till) <= self.inv_list[pos + 1],
+            Some(pos) => {
+                if let Some(x) = self.inv_list.get(pos + 1) {
+                    (till) <= x
+                } else {
+                    // This code should be unreachable because:
+                    // Inversion list query should not return out of bounds index
+                    false
+                }
+            }
```

Comment:
+1

Reply:
I originally put in this change to avoid the possibility of a panic. @sffc -- should that be a concern here?

OTOH, the comment asserts that it should be unreachable (based on the binary search result from the inversion list query not giving an out-of-bounds index). There is some precedent -- we used an `.unwrap()` in the CPT code because we couldn't avoid it and there weren't doubts about the code (which came from ICU4C).

Reply:
I do prefer what Elango has written over having unwraps in our code.  However, I would also be okay with an `unreachable!()`.

Reply:
Done.